README.md

```diff
@@ -289,7 +290,30 @@ java -Xmx8g -jar <path_to_hellbender_protected_jar>  PerformSegmentation  -S <sa
 ```
 
 
-##### Step 4. Call segments
+##### Step 4. Plot coverage profile
+
+###### Inputs
+- normalized coverage file (tsv) -- from step 2.
+- pre-normalized coverage file (tsv) -- from step 2.
+- segmented coverage file (seg) -- from step 3.
+- sample name
```

Comment:
Any constraints on the sample name in terms of characters? (eg whitespace, special chars etc) Does it need to be passed in double quotes?